hydra-node/src/Hydra/HeadLogic/Snapshot.hs

```diff
@@ -0,0 +1,89 @@
+module Hydra.HeadLogic.Snapshot where
```

Comment:
This module name is a confusing to the `Hydra.Snapshot` module holding the actual data structure.

Reply:
Renamed to `SnapshotOutcome`